lib/app_component.dart

```diff
@@ -380,7 +379,7 @@ class AppComponent {
       equationItems.add(totalValue);
 
       //this fixes precision issues sqrt(2)*sqrt(2) = 2.000000000004
-      if (totalValue.toString().contains(".")) {
+      if (totalValue.toString().contains(decimal)) {
```

Comment:
Hmm, I remember finding a similar class when trying to get around this. The problem was that It was based on an older version of dart and I think there were some compatibility issues. Also I might have had to refactor the entire project to use num instead of Dart (not sure about that though). If you can find it, link the 3rd part class you are referring to so I can make sure we're talking about the same one.